From fbb922cfcf4e389b0aa416d9a81c45f8f938f104 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Fri, 27 Feb 2015 13:39:35 -0500
Subject: [PATCH 170/315] systemd: import language into daemon environment

By default GDM is started with a pretty blank environment.

This commit makes sure that the default locale for the
system is used.

https://bugzilla.gnome.org/show_bug.cgi?id=745286
---
 data/Makefile.am    | 1 +
 data/gdm.service.in | 1 +
 2 files changed, 2 insertions(+)

diff --git a/data/Makefile.am b/data/Makefile.am
index cb59954f..7d521bbb 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -186,6 +186,7 @@ gdm.service: $(srcdir)/gdm.service.in
 	$(AM_V_GEN)sed \
 		-e 's,[@]sbindir[@],$(sbindir),g' \
 		-e 's,[@]GDM_INITIAL_VT[@],$(GDM_INITIAL_VT),g' \
+		-e 's,[@]LANG_CONFIG_FILE[@],$(LANG_CONFIG_FILE),g' \
 		-e 's,[@]PLYMOUTH_QUIT_SERVICE[@],$(PLYMOUTH_QUIT_SERVICE),g' \
 		-e 's, *$$,,g' \
 		< $< > $@
diff --git a/data/gdm.service.in b/data/gdm.service.in
index f7876e9c..ed74f219 100644
--- a/data/gdm.service.in
+++ b/data/gdm.service.in
@@ -10,6 +10,7 @@ IgnoreSIGPIPE=no
 BusName=org.gnome.DisplayManager
 StandardOutput=syslog
 StandardError=inherit
+EnvironmentFile=@LANG_CONFIG_FILE@
 
 [Install]
 Alias=display-manager.service
-- 
2.15.1

